src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -125,6 +125,8 @@ public final class Constants {
 
     public static final int FEEDBACK_DETAIL_TEXT_MAX_CHARACTERS = 5000;
 
+    public static final int LONG_FEEDBACK_MAX_LENGTH = 10_000_000;
```

Comment:
```suggestion
    /**
     * Arbitrary limit that is unlikely to be reached by real feedback in practice.
     * Avoids filling the DB with huge text blobs, e.g. in case an infinite loop in a test case outputs a lot of text.
     */
    public static final int LONG_FEEDBACK_MAX_LENGTH = 10_000_000;
```